src/main/java/roomescape/entity/repository/ReservationRepository.java

```diff
@@ -22,23 +23,28 @@ public ReservationRepository(JdbcTemplate jdbcTemplate, DataSource source) {
             .usingGeneratedKeyColumns("id");
     }
 
-    public List<Reservation> findAll() {
-        String sql = "SELECT * FROM reservation";
+    public List<ReservationOutDto> findAll() {
+        String sql = "SELECT \n" +
+            "    r.id as reservation_id, \n" +
+            "    r.name, \n" +
+            "    r.date, \n" +
+            "    t.id as time_id, \n" +
+            "    t.time as time_value \n" +
+            "FROM reservation as r inner join time as t on r.time_id = t.id\n";
```

Comment:
```suggestion
        String sql = """
            SELECT 
                r.id AS reservation_id,
                r.name,
                r.date,
                t.id AS time_id,
                t.time AS time_value
            FROM reservation AS r
            INNER JOIN time AS t ON r.time_id = t.id
            """;
          
```
아래 같이 수정하면 가독성이 조금 더 좋아질 것 같습니다.